wordpress-mobile/WordPress-Android
# Fix 13.8 translations

Re-applies https://github.com/wordpress-mobile/WordPress-Android/pull/10932 to the 13.8 branch prior to the release.

**To test:**
- Ensure that CI is green – this will tell us if we broke any strings files.
- Run these test cases in `WordPress/src/main/res` – the number `0` should be printed 7 times. This will tell us whether we caught all of the issues.

```
grep -r ' »<\/' .		| grep guten | wc -l
grep -r '»<\/' .		| grep guten | wc -l
grep -r '">"' .			| grep guten | wc -l
grep -r '”</string>' .	| grep guten | wc -l
grep -r '\"</string>' .	| grep guten | wc -l
grep -r '">%s”' .		| wc -l	# ensure we didn't break any strings starting with a quote
grep -r '"">' . 		| wc -l	# ensure we didn't add any double-closing quotes to resources
```
- Check the first changed line in each file to ensure that the find-and-replace didn't remove more characters than it should have.

PR submission checklist:
